src/frontend/templates/pages/Videos.js

```diff
@@ -9,7 +9,7 @@ export const Videos = () => {
     return (
     <main className='main'>
         <div className='videos-wrapper px-xs py-sm d-flex fw-wrap ai-c jc-c gap-1 mx-auto'>
-            { videos.map( (video,index) => <Video key={index} videoDetails={video} /> ) }
+            { videos.map( (video,index) => <Video fromHistory={false} key={index} videoDetails={video} /> ) }
```

Comment:
I've read it's not a good practice to use index as the key because the index of an element can change at any time.

Reply:
thanks for pointing that out, will fix this